Neos.ContentRepository.Core/Classes/Feature/DimensionSpaceAdjustment/DimensionSpaceCommandHandler.php

```diff
@@ -71,6 +71,8 @@ private function handleMoveDimensionSpacePoint(
         $streamName = ContentStreamEventStreamName::fromContentStreamId($command->contentStreamId)
             ->getEventStreamName();
 
+        // todo use WorkspaceName here as well but the command doesnt expose it
+        // https://github.com/neos/neos-development-collection/issues/4942
         self::requireDimensionSpacePointToBeEmptyInContentStream(
```

Comment:
Would be great to find out why https://github.com/neos/neos-development-collection/issues/4942 is how it is.
These were the only commands untouched by bernhards refactoring.